hazelcast/src/main/java/com/hazelcast/internal/config/ConfigValidator.java

```diff
@@ -263,6 +272,31 @@ public static void checkCacheConfig(InMemoryFormat inMemoryFormat, EvictionConfi
         }
     }
 
+    /**
+     * Checks {@link InMemoryFormat#NATIVE} data can be merged with the provided {@code mergePolicy}
+     */
+    private static void checkMergePolicySupportsNative(String name, String mergePolicy, InMemoryFormat inMemoryFormat) {
+        if (!getBuildInfo().isEnterprise() || inMemoryFormat != NATIVE) {
+            return;
+        }
+
+        try {
+            if (SplitBrainMergePolicy.class.isAssignableFrom(Class.forName(mergePolicy))) {
+                return;
+            }
+
+            String msg = "Split brain recovery is not supported for '%s'."
+                    + " Because its using merge policy `%s` to merge `%s` data."
```

Comment:
minor: `'%s', because it's` instead of `'%s'. Because its`?

Reply:
fixed